android/src/main/java/app/notifee/core/NotificationManager.java

```diff
@@ -410,31 +410,6 @@
         .continueWith(CACHED_THREAD_POOL, fullScreenActionContinuation);
   }
 
-  static Task<Void> cancelNotification(
```

Comment:
Is this being removed right? 😅

Reply:
Guessing so based on updated description 😀 so ignore me haha, just curious why this API was removed

Reply:
yea 😅  I just noticed it wasn't removed.  it snuck in again somehow 🙈